data/coalition/coalition missions.txt

```diff
@@ -1393,3 +1393,277 @@ mission "Arachi Cartoonist 2"
+			`	After a few minutes, bridges extend from just under the seating area of the next level above you to the circular platform in the middle of the room, and hundreds of Kimek contestants walk out upside-down, clinging to underside of the bridges and platform, as Heliarch agents divide them into two teams, numbering several hundred contestants each. The Heliarchs leave, and, after a long whistle is blown, the two teams of hanging Kimek charge at one another with surprising speed and agility, despite their swarming numbers and inverted orientation.`
+			`	The collective pile looks like somewhat like a collective sumo match as the teams attempt to dislodge their opponents from the platform. Within a few seconds, dozens of Kimek have been knocked or outright thrown to the net below, where they are helped out to the sides by workers. Looking through the net, you can see that this is being repeated on each floor. Another whistle blows after enough have fallen out, and the remaining Kimek stop fighting. Heliarch agents return to the ring and organize those left once more into two teams. They depart, and the whistle blows again. This time the fighters have more space to maneuver , so almost a minute passes before any fall.`
+			`	This process repeats several more times until finally, those that remain - less than a tenth of those that entered - are gathered in the center. As they leave the ring, without an announcement of any rankings or score, you become unsure if the game has ended.`
+			`	You feel something tap the top of your head and turn to find it is an Arach sitting behind you. "Confused, the human visitor is?" their little box translates.`
+			choice
+				`	"Yes. I have no idea what is going on."`
+				`	"I think I somewhat understand, but an explanation would be helpful."`
+			`	They breakdown the competition for you during what you learn is a brief break. "Numerous, the Kimek are, so many, many athletes competing, they have," they say. "Over a thousand, at first enter each floor. Redone, the teams are, once in half, that number is cut. The true game begins, when one hundred remain."`
+			`	As if on cue, the remaining contestants reenter the ring, this time wearing what looks like some kind of goggles, perhaps as protective gear. This time, instead of being split into two teams to fight, they are divided into pairs, and each pair moves to one of the small rings painted on the platform.`
```

Comment:
```suggestion
			`	Just then, the remaining contestants reenter the ring, almost as if on queue, now wearing what looks like some kind of goggles, perhaps as protective gear. This time, instead of being split into two teams to fight, they are divided into pairs, and each pair moves to one of the small rings painted on the platform.`
```